django_mongodb/features.py

```diff
@@ -61,6 +61,17 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         "aggregation.tests.AggregateTestCase.test_aggregation_default_passed_another_aggregate",
         "aggregation.tests.AggregateTestCase.test_annotation_expressions",
         "aggregation.tests.AggregateTestCase.test_reverse_fkey_annotate",
+        "aggregation_regress.tests.AggregationTests.test_annotation_disjunction",
+        "aggregation_regress.tests.AggregationTests.test_decimal_aggregate_annotation_filter",
+        # QuerySet.extra(select=...) should raise NotSupportedError instead of:
+        # 'RawSQL' object has no attribute 'as_mql'.
+        "aggregation_regress.tests.AggregationTests.test_annotate_with_extra",
+        "aggregation_regress.tests.AggregationTests.test_annotation",
+        "aggregation_regress.tests.AggregationTests.test_more_more3",
+        "aggregation_regress.tests.AggregationTests.test_more_more_more3",
+        # To triage:
```

Comment:
Here!

Reply:
Great tests, in the first one we have two issues: 

1. It is related with joins.
2. the way that we are storing object_ids in Django models. we cannot store a char[24].

Note that : `collection.find({'_id': '66c42726db971422470255ed'})` it won't match the thing that have the id `ObjectId('66c42726db971422470255ed')`.

Reply:
the second test is fixed, just another collision. Fixed in the last commit and test removed from features.py

Reply:
I think we should open a new issue for the first one. It's out of aggregation's scope.

Reply:
Thanks. I squashed the third commit with the second since it was actually a regression in that commit.